src/component/InsertBox.js

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback, useState } from "react";
```

Comment:
React 17 부터는 `import React from 'react';`를 더이상 적지 않아도 된다고 하더라구요! [참고링크](https://ui-tricks.tistory.com/50) 그 전에 습관이 들어서 저도 무의식적으로 적었었는데, 오히려 적지 않는 것이 더 도움이 된다고 해서 저도 요즘에는 빼고 적으려고 노력하구있어요!

```suggestion
import { useCallback, useState } from "react";
```

Reply:
헉 저도 이건 몰랐네요!!! 좋은 정보 감사합니당👏🏻👏🏻

Reply:
아하 그렇군요!! 다음부터는 그렇게 해보아야겠어요! 꿀팁 정말 감사합니다 문기님👍👍